daemon/environment.cpp

```diff
@@ -563,7 +566,9 @@ pid_t start_install_environment(const std::string &basename, const std::string &
 
     argv[4] = strdup("-");
     argv[5] = 0;
-    _exit(execv(argv[0], argv));
+    execv(argv[0], argv);
+    log_perror("execv failed");
+    _exit(142);
```

Comment:
I can change it, but ultimately it doesn't matter, anything non-zero is an error are we cannot tell which exit code is from this error vs an error code from tar.